addons/medical_damage/functions/fnc_woundsHandlerSQF.sqf

```diff
@@ -93,17 +93,24 @@ private _bodyPartVisParams = [_unit, false, false, false, false]; // params arra
             _bodyPartVisParams set [[1,2,3,3,4,4] select _bodyPartNToAdd, true]; // Mark the body part index needs updating
 
 
-            // The higher the nastiness likelihood the higher the change to get a painful and bloody wound
-            private _nastinessLikelihood = linearConversion [0, 20, (_woundDamage / _thresholdWoundCount), 0.5, 30, true];
-            private _bleedingModifier = 0.25 + 8 * exp ((random [-4.5, -5, -6]) / _nastinessLikelihood);
-            private _painModifier = 0.05 + 2 * exp (-2 / _nastinessLikelihood);
+            // Damage to limbs/head is scaled higher than torso by engine
+            // Anything above this value is guaranteed worst wound possible
+            private _worstDamage = [2, 1, 4, 4, 4, 4] select _bodyPartNToAdd;
 
-            private _bleeding = _injuryBleedingRate * _bleedingModifier;
+            // More wounds means more likely to get nasty wound
+            private _countModifier = 1 + random(_i - 1);
```

Comment:
I'm not sure I understand this line, if I test multi hit wound like
`[cursorTarget, 0.4, "body", "shell", player] call ace_medical_fnc_addDamageToUnit`
I get
```
test: _i=1, _countModifier=1 z\ace\addons\medical_damage\functions\fnc_woundsHandlerSQF.sqf:102
test: _i=2, _countModifier=1.44389 z\ace\addons\medical_damage\functions\fnc_woundsHandlerSQF.sqf:102
test: _i=3, _countModifier=1.91001 z\ace\addons\medical_damage\functions\fnc_woundsHandlerSQF.sqf:102
test: _i=4, _countModifier=3.46463 z\ace\addons\medical_damage\functions\fnc_woundsHandlerSQF.sqf:102
test: _i=5, _countModifier=2.4834 z\ace\addons\medical_damage\functions\fnc_woundsHandlerSQF.sqf:102
```
if we create 5 random wounds, why do the later ones bleed more?

Reply:
It's basically just making it so that if you receive multiple wounds all at once, subsequent wounds have a higher chance of being worse.